core/survey/_survey/surveyNodeDefsLayout.js

```diff
@@ -0,0 +1,47 @@
+import * as R from 'ramda'
+
+import { uuidv4 } from '@core/uuid'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+import * as SurveyNodeDefs from './surveyNodeDefs'
+
+const _updateRenderType = ({ survey, surveyCycleKey, nodeDef, renderType, nodeDefLayout }) => {
+  if (renderType === NodeDefLayout.renderType.table) {
+    // Assoc layout children
+    const nodeDefChildren = SurveyNodeDefs.getNodeDefChildren(nodeDef)(survey)
+    return NodeDefLayout.assocLayoutChildren(surveyCycleKey, R.map(NodeDef.getUuid, nodeDefChildren))(nodeDefLayout)
+  }
+
+  if (renderType === NodeDefLayout.renderType.form) {
+    const nodeDefLayoutUpdated = NodeDefLayout.dissocLayoutChildren(surveyCycleKey)(nodeDefLayout)
+    // Entity rendered as form can only exists in its own page
+    if (NodeDefLayout.isDisplayInParentPage(surveyCycleKey)(nodeDef)) {
+      return NodeDefLayout.assocPageUuid(surveyCycleKey, uuidv4())(nodeDefLayoutUpdated)
+    }
+    return nodeDefLayoutUpdated
+  }
+
+  return nodeDefLayout
+}
+
+const _isEntityAndKeyRenderMode = ({ nodeDef, key }) =>
```

Comment:
useless function

Reply:
it makes the code a bit more readable and avoids a comment in that line, but I can remove it, of course.

Reply:
it makes it less readable. if a function or variable is one line and used only ones, then you realize that it serves no purpose